src/views/Home/components/Mission/mission.scss

```diff
@@ -1,3 +1,108 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/responsiveness";
+@use "~stylesheets/typography";
+@use "~stylesheets/variables";
 
-/* Delete these lines when you begin development */
+.Mission {
+  position: relative;
+
+  /* Text Layer */
+  &--text {
+    display: flex;
+    flex-direction: column;
+    background: url("~images/mission-background.png");
+    background-size: 100%;
+    padding: 10% 22%;
+  }
+
+  &__section {
+    z-index: 2;
+
+    &:first-of-type {
+      display: flex;
+      margin-bottom: 13%;
+    }
+
+    &--subsection {
+      max-width: 100%;
+    }
+  }
+
+  &__title {
+    @include typography.title-body;
+    padding: var(--medium-spacer) var(--extra-extra-large-spacer);
+
+    background: rgba(196, 219, 255, 0.57);
+    mix-blend-mode: hard-light;
+    backdrop-filter: blur(4px);
+    border-radius: 58px 58px 0 0;
+  }
+
+  &__body {
+    display: flex;
+    padding: var(--medium-spacer) var(--large-spacer) var(--medium-spacer) var(--extra-extra-large-spacer);
+
+    // Variable names and styling to be resolved after other pull requests.
+    background: linear-gradient(180deg, #7c9bff 0%, rgba(138, 157, 218, 0.6) 100%);
+    box-shadow: 0 0 22px -3px #a687ff;
+    border-radius: 0 0 63px 63px;
+  }
+
+  &__rectangle {
+    width: 5px;
+    flex-shrink: 0;
+    margin-right: var(--medium-spacer);
+
+    background: var(--dark-navy);
+    border-radius: 155px;
+  }
+
+  &__highlight {
+    width: 75px;
+    flex-shrink: 0;
+    margin: 0 -65px 0 -10px;
+
+    // Variable names and styling to be resolved after other pull requests.
+    background: linear-gradient(180deg, #fc3b3b 0%, #1a49e1 100%);
+    box-shadow: 0 0 22px -3px #a687ff;
+    border-radius: 59px;
+  }
+
+  &__text {
+    @include typography.description-dark;
+    line-height: 105%;
+  }
+
+  /* Image Layer */
+  &--images {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 0;
+  }
+
+  &__can--left {
+    position: absolute;
+    width: 18%;
+    left: 1%;
+    top: 0;
+  }
+
+  &__can--right {
+    position: absolute;
+    width: 18%;
+    left: 80%;
+    top: 32%;
+  }
+
+  &__flower {
+    position: absolute;
+    width: 28%;
+    left: 5%;
+    top: 43%;
```

Comment:
consider throwing in a relative unit here like `vh` using the `calc` function! just using it on `top` puts the flower in places with better contrast across mobile & desktop.
![Screenshot from 2020-12-19 13-29-53](https://user-images.githubusercontent.com/19658882/102700012-50524680-41fe-11eb-934a-a1d2c7576c79.png)

Reply:
also consider revising these columns so they're wider & friendlier to read at this breakpoint!

Reply:
I went ahead an used `bottom` instead - let me know how it looks!